renaming_aps/README.md

```diff
@@ -0,0 +1,78 @@
+# Renaming APs
+This is a Python script that uses the [Pycentral](https://pypi.org/project/pycentral/) library to achieve the following steps on an Aruba Central account- 
+1. Copy device configuration
```

Comment:
I would use 'AP' instead of 'device' in the Readme as this workflow is applicable for APs & not other device types